pkgs/overlay.nix

```diff
@@ -52,6 +53,37 @@ builtins.mapAttrs (_: patchPhps (fetchpatch {
   })) {
     inherit (super) php83 php84;
   }
+# temporarily added to overlay to ensure it is built by hydra.
+# TODO: drop in 25.05
+# Use curl 8.4.0 for PHP
+# FC-40106, FC-40100
+// lib.listToAttrs (map (v: lib.nameValuePair "php${v}-FC-40106" (
+  super."php${v}".override {
+          packageOverrides = final: prev: {
+            extensions = prev.extensions // {
+              curl = prev.extensions.curl.overrideAttrs (_: let
+                curl_8_4 = self.curl.overrideAttrs (_: {
+                  version = "8.4.0";
+                  name = "curl-8.4.0";
```

Comment:
This should not be needed, as set be pname+version

Reply:
I have copied that code in verbatim from the affected customer project. But as removing this should not change the resulting derivation, i can adjust that as well.